src/views/Players.vue

```diff
@@ -270,7 +270,7 @@ export default {
       query: QUERY_GET_CALCULATED_SCORES,
       variables() {
         return {
-          leagueId: this.id.toString(),
+          leagueId: this.id,
```

Comment:
Does this need to be a number?

Reply:
Strangely this complex query needs the value to be string. I updated it to make sure it is being toStringed again